### PR TITLE
DM-27844: exposurelog: add summit deployment and updated NFS deployment

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

**Review by afausti — APPROVED** [review]
Looks great!

**Review by afausti — CHANGES_REQUESTED** [review]
Actually make sure you have the correct host and vault_path for the summit.

File: services/exposurelog/values-summit.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  nfs_server_1: comcam-arctl01.cp.lsst.org
+  butler_uri_1: /volume_1  # a guess
+
+  ingress:
+    enabled: true
+    annotations: {}
+    #   kubernetes.io/ingress.class: nginx
+    #   kubernetes.io/tls-acme: "true"
+    hosts:
+      - host: lsst-nts-k8s.ncsa.illinois.edu
```
Comment (afausti) [inline services/exposurelog/values-summit.yaml:13]: That should be summit-lsp.lsst.codes

File: services/exposurelog/values-summit.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        paths:
+          - /exposurelog
+    tls: []
+    #  - secretName: chart-example-tls
+    #    hosts:
+    #      - chart-example.local
+
+vault_path: secret/k8s_operator/lsst-nts-k8s.ncsa.illinois.edu/postgres
```
Comment (afausti) [inline services/exposurelog/values-summit.yaml:21]: and the vault_path should be secret/k8s_operator/summit-lsp.lsst.codes/postgres and I confirm that the exposurelog_password is there.

File: services/exposurelog/values-summit.yaml (hunk 1 above)
Comment (r-owen) [reply services/exposurelog/values-summit.yaml:13]: Thank you for catching that and for the corrected value.

File: services/exposurelog/values-summit.yaml (hunk 2 above)
Comment (r-owen) [reply services/exposurelog/values-summit.yaml]: DItto

**Comment by r-owen** [comment]
I really appreciate the corrected secrets values. Updated as per your suggestion.

**Review by afausti — APPROVED** [review]
Look great!

**Pull request merged**